app/js/main.js

```diff
@@ -254,19 +250,42 @@ function startup() {
     // Listen for leaving frame rate input
     inputChangeFR.addEventListener("blur", function() {
         inputChangeFR.value = frameRate;
-        if(inputChangeFR.value > 60 || inputChangeFR.value < 1 || NaN || inputChangeFR.length > 2) {
+        if (inputChangeFR.value > 60 || inputChangeFR.value < 1 || NaN || inputChangeFR.length > 2) {
             inputChangeFR.value = 15;
         }
     });
 
     // Toggle capture and playback windows
     btnLiveView.addEventListener("click", function() {
-        if (winMode === "capture") {
-            winMode = "playback";
-        } else if (winMode === "playback") {
-            winMode = "capture";
+        // Switch from frame preview back to live view
+        if (document.querySelector(".frame-reel-preview.selected")) {
+            document.querySelector(".frame-reel-preview.selected").classList.remove("selected");
+            videoStop();
+        }
+
+        switchMode("capture");
+    });
+
+    // Preview a captured frame
+    document.querySelector("#reel-captured-imgs").addEventListener("click", function(e) {
```

Comment:
Because I did not know that variable existed and I didn't Ctrl+f the selector. Will fix.